R/RunWorkflows.R

```diff
@@ -39,12 +41,24 @@ RunWorkflows <- function(
 
   lResults <- list()
   for (wf in lWorkflows) {
-    lResult <- RunWorkflow(
-      lWorkflow = wf,
-      lData = c(lResults, lData),
-      lConfig = lConfig,
-      bReturnResult = bReturnResult,
-      bKeepInputData = bKeepInputData
+    lResult <- tryCatch(
+      {
+        RunWorkflow(
+          lWorkflow = wf,
+          lData = c(lResults, lData),
+          lConfig = lConfig,
+          bReturnResult = bReturnResult,
+          bKeepInputData = bKeepInputData
+        )
+      },
+      error = function(e) {
+        if(bLogFailAsError) {
```

Comment:
@samussiah was this what you had in mind about the parameterization of the logging aspect? I turned it into an argument `bLogFailsAsError` that toggles TRUE/FALSE for Error/Warn level of the log